src/app/containers/ATIAnalytics/index.jsx

```diff
@@ -5,7 +5,7 @@ import AmpATIAnalytics from './amp';
 import getArticleAtiParams from './params/article';
 import getFrontPageAtiParams from './params/frontpage';
 import getMediaPageAtiParams from './params/media';
-import { pageDataPropType } from '#models/propTypes/data';
+import { pageDataPropType } from '#models/propTypes/data/index';
```

Comment:
in this file, when I highlight and right-click on `pageDataPropType`  then select "Go to definition" it goes to the file where `pageDataPropType` was declared but when I do it for the other imports especially the relative ones, that doesn't seem to work @FK78 could you check this out ??